pkg/monitor/cluster/clustervmpower.go

```diff
@@ -0,0 +1,38 @@
+package cluster
+
+import (
+	"context"
+
+	"github.com/Azure/ARO-RP/pkg/util/stringutils"
+	"github.com/Azure/ARO-RP/pkg/util/virtualmachines"
+)
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+func (mon *Monitor) emitVMPowerStatus(ctx context.Context) (stopped bool, err error) {
+	resourceGroupName := stringutils.LastTokenByte(mon.oc.Properties.ClusterProfile.ResourceGroupID, '/')
+	stoppedVMs, err := virtualmachines.ListStopped(ctx, mon.vmClient, resourceGroupName)
+	if err != nil {
+		return
+	}
+
+	if len(stoppedVMs) > 0 {
+		stopped = true
+		// 1. Add two statuses to cluster:
+		//    a. There are stopped VMs right now
+		//    b. There were stopped VMs recently
+		//    (After the VM is on again, b. will warn customer. Acknowledgement will clear b.)
```

Comment:
I think start off simply, and just emit a metric (and return early like you are doing)